forms-flow-api/src/formsflow_api/resources/application.py

```diff
@@ -363,3 +364,62 @@ def get():
             )
         except BusinessException as err:
             return err.error, err.status_code
+
+
+@cors_preflight("POST,OPTIONS")
+@API.route("/create-application", methods=["POST", "OPTIONS"])
+class ApplicationCreation(Resource):
+    """Resource for application creation."""
+
+    @staticmethod
+    @auth.require
+    @profiletime
+    def post():
+        """Post a new application using the request body.
+
+        : data: form submission data
+        : formId:- Unique Id for the corresponding form
+        : submissionId:- Unique Id for the submitted form
+        : formUrl:- Unique URL for the submitted application
+        """
+        application_json = request.get_json()
+        data = request.get_json()
+
+        try:
+            application_schema = ApplicationSchema()
+            dict_data = application_schema.load(application_json)
+            formio_service = FormioService()
+            form_io_token = formio_service.get_formio_access_token()
+            formio_service.post_submission(data, form_io_token)
```

Comment:
so need to have the following flow
1. post the submission to Formio.
2. wait for the response if it's successful create the application with `submissionId` from the response else handle the exception and respond to the client.
3. If both submissions are successful respond back to the client with a success response.